### PR TITLE
Add network policy

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/4033

### CODE REVIEW

**Comment by tomahawk28** [comment]
Don't you need to update helm/chart-operator-chart/templates/rbac.yaml?

**Comment by corest** [comment]
@tomahawk28 nope. For what purpose?

**Comment by tomahawk28** [comment]
I thought I need to list resource type in rbac if you want to create it. Maybe that's not true in the initial setup, sorry.

**Comment by corest** [comment]
Network policy installed by tiller, in the same way, as for other apps. And tiller has `cluster-admin` role. Am I missing something here?

**Comment by corest** [comment]
But the thing you've mentioned may apply here https://github.com/giantswarm/helmclient/pull/106 as chart-operator in control-plane needs to install tiller in tenant cluster

File: helm/chart-operator-chart/Chart.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart for the chart-operator
 name: chart-operator-chart
-version: 0.7.1
+version: 0.8.0
```
Comment (rossf7) [inline helm/chart-operator-chart/Chart.yaml]: @corest Good to bump the versions here. If you haven't already you also need to bump the release channel in the WIP version of cluster-operator.

https://github.com/giantswarm/cluster-operator/blob/4161b4dfdcb02b73c4574d1c6f9335ac0f514587/pkg/v17/resource/chartoperator/resource.go#L27

Comment (corest) [reply helm/chart-operator-chart/Chart.yaml]: I'll bump version in cluster-operator later as there is another change expected to arrive here first

**Pull request merged**